assets/build/install.sh

```diff
@@ -34,7 +34,7 @@ paxctl -Cm `which nodejs`
 rm -rf /etc/ssh/ssh_host_*_key /etc/ssh/ssh_host_*_key.pub
 
 # add ${GITLAB_USER} user
-adduser --disabled-login --gecos 'GitLab' ${GITLAB_USER}
+adduser --disabled-login --gecos 'GitLab' ${GITLAB_USER} --home ${GITLAB_HOME}
```

Comment:
Without this adduser creates /home/$GITLAB_USER by default but everything else points to /home/$GITLAB_HOME